apps/condo/domains/notification/adapters/firebaseAdapter.js

```diff
@@ -0,0 +1,200 @@
+const admin = require('firebase-admin')
+const isEmpty = require('lodash/isEmpty')
+const isNull = require('lodash/isNull')
+const get = require('lodash/get')
+const faker = require('faker')
+
+const {
+    PUSH_FAKE_TOKEN_SUCCESS,
+    PUSH_FAKE_TOKEN_FAIL,
+    FIREBASE_CONFIG_ENV,
+} = require('../constants/constants')
```

Comment:
Looks weird

Reply:
Can you please tell some details? I can't catch the idea.